flask_classful.py

```diff
@@ -501,8 +501,8 @@ def get_true_argspec(method):
 
 def unpack(value):
     """Return a three tuple of data, code, and headers"""
-    if not isinstance(value, tuple):
-        return value, 200, {}
+    # if not isinstance(value, tuple):
```

Comment:
don't uncomment this as the function has support for not tuple value case.

Reply:
they already checked it before, line `304`:
```
if isinstance(response, tuple):
                response, code, headers = unpack(response)
```

Reply:
they're different. this is what the function provides and that is how the function is used. The provided function cannot know how it is used in different use cases. For example, now I have another location where the provided function is used without tuple check, the provided function with your changes will be crashed.